content/en/digital_certificate.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+A digital certificate or  a public key certificate, which is issued by the Certificate Authority, is an electronic password that allows an entity to exchange data or resources over the internet in a secured manner. This is achieved by proving the authenticity of the user through the use of encryption methods and set of roles, policies, resources used to manage, use and revoke this digital certificate and encryptions. Much as a passport certifies one’s identity, the purpose of a digital certificate is to establish the identity of users within the ecosystem.
+
+## Problem it addresses
+Digital certificates help solve web-based communication security issues. Networks developed without much thought to security. TCP/IP, the most prevalent communication protocol, was designed around precepts of robust communication rather than secure transactions. While the OSI network model designed a layer between the application and network that could manage security handshakes, TCP/IP left everything above the communication layer to the application. Consequently each application developed its own negotiation strategy. Rather than building a generic, reusable security mechanism, applications relied on proprietary or embedded techniques for shuffling account information across the network to the platform or application for validation.
```

Comment:
This section here is far too technical. Please remember that we are trying to make this accessible to non-technical people as well. TCP/IP, OSI network model, security handshakes, layers between apps and networks...these are all things that aren't commonly understood. Could please try to explain this without all these industry terms?

Reply:
Thanks very much @CathPag  for the feedback. I revised my work and made some updates to it